tests/cypress/support/commands.ts

```diff
@@ -44,13 +44,26 @@ Cypress.Commands.add('gitRepoAuth', (gitAuthType, userOrPublicKey, pwdOrPrivateK
 });
 
 
-// Command add Fleet Git Repository
-Cypress.Commands.add('addFleetGitRepo', ({ repoName, repoUrl, branch, path, gitAuthType, userOrPublicKey, pwdOrPrivateKey, keepResources }) => {
-  cy.clickButton('Add Repository');
-  cy.contains('Git Repo:').should('be.visible');
-  cy.typeValue('Name', repoName);
-  cy.typeValue('Repository URL', repoUrl);
-  cy.typeValue('Branch Name', branch);
+// Command add and edit Fleet Git Repository
+// TODO: Rename this command name to 'addEditFleetGitRepo'
+Cypress.Commands.add('addFleetGitRepo', ({ repoName, repoUrl, branch, path, gitAuthType, userOrPublicKey, pwdOrPrivateKey, keepResources, correctDrift, edit=false }) => {
+  cy.accesMenuSelection('Continuous Delivery', 'Git Repos');
+  if (edit === true) {
+    // Change Namespace from local to default when interacting with another namespace.
```

Comment:
Sorry, I'm confused, from local to default or the other way around?

Reply:
I will remove this comment in next patchset.